angular js session 2 assignment/products/src/app/app.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="jumbotron">
+  <h2 class="text-center">{{title}}</h2>
+</div>
+<div class="main-content">
+  <div class="container">
+    <div class="row">
+      <div class="col-lg-3 col-md-4 col-sm-6 col-xs-12" *ngFor="let product of products">
+        <div class="card card-outline-primary mb-3 text-center">
+          <div class="card-block">
+            <h3 class="card-title">{{product.name}}</h3>
+            <p class="card-text">Product ID: {{product.id}}</p>
+            <span>Rs. {{product.price}}</span>
```

Comment:
Instead of hard coding currency symbol, it should also be available in component.